backend/django_backend/pong/consumers.py

```diff
@@ -34,51 +33,46 @@ async def connect(self):
         if not self.pong_game.room_group_name:
             self.pong_game.add_room_group_name(self.room_group_name)
 
+        # Add player to game
         if not self.pong_game.player1:
             self.player = Player(1, "Player 1")
             self.pong_game.add_player(self.player)
-            self.thread["player1"] = True
         elif not self.pong_game.player2:
```

Comment:
For RushMaverick: You can remove the line 40 when you are working on your threejs ;)

Reply:
Is this because we are not using a player2 in that moment?

Reply:
Both players are working but then you don't have to open two browsers to test. Plus currently both players have to disconnect before you can start a new game with the same name.